components/organisms/ImageBox.js

```diff
@@ -0,0 +1,50 @@
+import propTypes from 'prop-types'
+
+import Image from 'next/image'
+
+import SearchCard from '../molecules/SearchCard'
+
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+/**
+ *
+ */
+export default function ImageBox(props) {
+  const t = props.locale === 'en' ? fr : en
+
+  return (
+    <div className=" bg-deep-blue-solid">
+      <div
+        className="flex flex-col md:flex-row  bg-deep-blue-solid  md:container mx-auto md:pl-6 md:px-2"
+        id="image-box"
+        data-testid="image-box"
+      >
+        <SearchCard
+          lang={props.locale}
+          headerText={t.searchFindBenefits}
+          paraText={t.searchDesc}
+          viewBenefitsServices={t.searchViewAllBenefits}
+          searchBarPlaceholder={t.searchPlaceholder}
+          searchBarText={t.search}
+          onSubmitHref="/searchResult"
+        />
+        <div className="aspect-w-5 aspect-h-2 md:aspect-h-1 xl:ml-3 w-full min-w-max mr-break-out">
+          <Image
+            src={
+              'https://www.canada.ca/content/dam/decd-endc/images/clear-lake-snowy-mountain.png'
```

Comment:
Can we pass Image src in as a prop so we can reuse components and add tests for the the component?
Also, we might want to consider passing in a component in the future instead of using SearchCard directly for reuse